src/live.asp.net/TagHelpers/ScriptStyleInliningTagHelper.cs

```diff
@@ -12,11 +12,12 @@
 namespace live.asp.net.TagHelpers
 {
     [TargetElement("script", Attributes = "inline")]
-    public class ScriptInliningTagHelper : TagHelper
+    [TargetElement("style", Attributes = "inline")]
+    public class ScriptStyleInliningTagHelper : TagHelper
     {
         private readonly IFileProvider _wwwroot;
 
-        public ScriptInliningTagHelper(IHostingEnvironment env)
+        public ScriptStyleInliningTagHelper(IHostingEnvironment env)
```

Comment:
Did you actually test this? There should be a change to the layout page to actually enable inlining the `site.css` file, and you'll probably have to change the Tag Helper to look at the `href` attribute as well because that's what stylesheet `<link />` tags use, not `src`.